internal/services/streamanalytics/stream_analytics_job_resource.go

```diff
@@ -102,6 +102,17 @@ func resourceStreamAnalyticsJob() *pluginsdk.Resource {
 				Default: string(streamanalytics.EventsOutOfOrderPolicyAdjust),
 			},
 
+			"job_type": {
```

Comment:
as job is in the resource name should we make this
```suggestion
			"type": {
```

otherwise it'll be `azurerm_stream_analyitics_job.job_type`

Reply:
Updated